docs/topics/api/hero.rst

```diff
@@ -41,8 +41,8 @@ small number of shelves this endpoint is not paginated.
     :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
     :>json array results: The array containing the results for this query.
     :>json object results[].gradient: The background colors used for the gradient.
-    :>json string results[].gradient.start: The starting color for gradient - typically top or left.
-    :>json string results[].gradient.end: The ending color for gradient - typically bottom or right.
+    :>json string results[].gradient.start: The starting color name for gradient - typically top or left. The names are the photon color variables from https://github.com/FirefoxUX/photon-colors/blob/master/photon-colors.scss
```

Comment:
Nit: Should be "the name is from", not "the names are", as this is listed once for each name.